dm/dm-ansible/scripts/dm.json

```diff
@@ -5343,6 +5343,94 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "Is waiting shard DDL lock to be resolved, >0 means waiting",
```

Comment:
🤔

Reply:
fixed in https://github.com/pingcap/dm/pull/853/commits/eb37c535c53161709060c2916d17887948692735